docs/source/_static/css/parlai_theme.css

```diff
@@ -16,7 +16,7 @@ h1, h2, .rst-content .toctree-wrapper p.caption, h3, h4, h5, h6, legend, p.capti
 }
 
 nav p.caption {
-    color: #fcfcfc;
+    color: #fcfcfc !important;
```

Comment:
I would recommend getting rid of the `!important`. You can do what you're trying to achieve by making the selector here [more specific](https://specificity.keegan.st/), e.g. something like **`.wy-menu.wy-menu-vertical p.caption`** should beat out the other selector adding the blue font color (`.wy-menu-vertical p.caption`)